src/main/haskell/kore/src/Kore/Builtin/Set.hs

```diff
@@ -294,16 +299,43 @@ evalDifference =
             rightIdentity <|> bothConcrete
         )
 
+evalToList :: Builtin.Function
+evalToList = Builtin.functionEvaluator evalToList0
+  where
+    evalToList0
+        :: Ord (variable Object)
+        => MetadataTools Object StepperAttributes
+        -> StepPatternSimplifier Object variable
+        -> Sort Object
+        -> [StepPattern Object variable]
+        -> Simplifier (AttemptedFunction Object variable)
+    evalToList0 tools _ resultSort arguments =
+        Builtin.getAttemptedFunction $ do
+            let _set =
+                        case arguments of
+                            [_set] -> _set
+                            _      -> Builtin.wrongArity toListKey
+            _set <- expectBuiltinSet toListKey tools _set
+            Builtin.appliedFunction
```

Comment:
There is a function [`returnList`](https://github.com/kframework/kore/blob/master/src/main/haskell/kore/src/Kore/Builtin/List.hs#L145) which you could export from `Kore.Builtin.List`:
```.hs
returnList . Seq.fromList . Set.toList $ _set
```

Reply:
I didn't think I can do that because I needed to call `fromConcretePurePattern`, but I can do this instead:

```Haskell
List.returnList resultSort
    . fmap Kore.fromConcretePurePattern
    . Seq.fromList
    . Set.toList
    $ _set
```